packages/cf-deploy-config-writer/test/unit/index-app.test.ts

```diff
@@ -71,56 +71,53 @@ describe('CF Writer App', () => {
         jest.resetAllMocks();
     });
 
-    describe('Generate HTML5 App Config', () => {
```

Comment:
Just curious was this describe block causing the hooks not to be ran as expected?

Reply:
I dont! I just know from past experience around timeouts, that nested describes don't act nicely! 

This might not be the root cause but I cant think of anything else.